conf/locale/zh_CN/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,12905 @@
+# #-#-#-#-#  tos.po (stanford-openedx)  #-#-#-#-#
+# Stanford OpenEdX translation file
+# Copyright (C) 2018 Stanford University
+# This file is distributed under the GNU AFFERO GENERAL PUBLIC LICENSE.
+# Stanford OpenEdX Team, 2018
+# 
+# Translators:
```

Comment:
Why are there so many `Translators` stanzas?

Reply:
When merged with msgcat, the file headers just get joined as is